mk/build_optee_for_android.mk

```diff
@@ -0,0 +1,88 @@
+##########################################################
```

Comment:
Renamed to aosp_optee.mk. 
As a common mk file to be included somewhere, I did not see such file named as Android.mk in the AOSP.
And naming it to Android.mk will make it picked up by default, but if there is the case that user does not want to build OP-TEE components with the OP-TEE projects synced there, then this is not what we want, It's better to have this file included only when necessary I think.